hub/saml-proxy/src/main/java/uk/gov/ida/hub/samlproxy/controllogic/SamlMessageSenderHandler.java

```diff
@@ -101,7 +101,7 @@ public SamlMessage generateErrorResponseFromHub(final SessionId sessionId, Strin
         AuthnResponseFromHubContainerDto authnResponseFromHub = sessionProxy.getErrorResponseFromHub(sessionId);
         Response samlResponse = responseTransformer.apply(authnResponseFromHub.getSamlResponse());
         validateAndLogSamlResponseSignature(samlResponse);
-        SamlMessage samlMessage = new SamlMessage(authnResponseFromHub.getSamlResponse(), SamlMessageType.SAML_RESPONSE, authnResponseFromHub.getRelayState(), authnResponseFromHub.getPostEndpoint().toString(), Optional.<Boolean>absent());
+        SamlMessage samlMessage = new SamlMessage(authnResponseFromHub.getSamlResponse(), SamlMessageType.SAML_RESPONSE, authnResponseFromHub.getRelayState(), authnResponseFromHub.getPostEndpoint().toString(), Optional.<Boolean>empty());
```

Comment:
`<Boolean>` seems to have been removed everywhere else but here?

Reply:
Good spot Tom. Let me change that.